dialects/linux/dproc.c

```diff
@@ -1518,11 +1552,44 @@ process_proc_map(p, s, ss)
 		efs = sv = 1;
 	    else
 		efs = 0;
-	    if (!efs) {
-		if (HasNFS)
-		    sv = statsafely(fp[6], &sb);
-		else
-		    sv = stat(fp[6], &sb);
+
+	    /* For prcesses in different mount namespace from lsof process,
```

Comment:
```suggestion
	    /* For processes in different mount namespace from lsof process,
```

Reply:
@Hunter1016 I think you missed this one …

Reply:
Ah, I've forced-updated. Thanks a lot. @BenBE